test/api_integration/apis/core/index.js

```diff
@@ -22,19 +22,6 @@ export default function ({ getService }) {
   const supertest = getService('supertest');
 
   describe('core', () => {
-    describe('request context', () => {
-      it('provides access to elasticsearch', async () =>
-        await supertest.get('/requestcontext/elasticsearch').expect(200, 'Elasticsearch: true'));
-
-      it('provides access to SavedObjects client', async () =>
-        await supertest
-          .get('/requestcontext/savedobjectsclient')
-          .expect(
-            200,
-            'SavedObjects client: {"page":1,"per_page":20,"total":0,"saved_objects":[]}'
-          ));
-    });
```

Comment:
So, these two tests are based on endpoints that were living in the KP testbed plugin.

I just removed them as I didn't feel like they were adding much value now that most of the plugins have been migrated and using the ES and SO clients from the context.

If we prefer, I can create a test plugin with the testbed's endpoints instead.